lms/services/blackboard_api/service.py

```diff
@@ -59,6 +65,24 @@ def list_files(self, course_id):
             schema=BlackboardListFilesSchema,
         ).validated_data
 
+    def public_url(self, course_id, document_url):
+        # Get just the file_id part out of the
+        # blackboard://content-resource/file_id/ URL.
+        file_id = re.search(
+            r"blackboard:\/\/content-resource\/(?P<file_id>[^\/]*)\/", document_url
+        )["file_id"]
```

Comment:
I think the naming here is great, "document_url" vs "file_id" I think we should follow the same or simlar naming convention on `_FileSchema`